packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,240 @@
+            setCities(allCities.filter((city) => city.includes(inputValue)));
+          } else {
+            setCities(allCities);
+          }
+          setCurrentInputValue(inputValue);
+        }}
+        // Selected Items in AutoComplete
+        selected={currentSelections}
+        onSelectChange={({ values }) => {
+          setCurrentSelections(values);
+        }}
+      />
+      <DropdownOverlay>
+        <ActionList>
+          {cities.map((city) => (
+            <ActionListItem
+              title={city}
+              value={city.toLowerCase()}
+            />
+          ))}
+        </ActionList>
+      </DropdownOverlay>
+    </Dropdown>
+  );
+};
+```
+
+</details>
+
+### In BottomSheet
+
+> **Warning**
+>
+> We will have to turn BottomSheetHeader (and maybe BaseHeader) to a slot to allow adding Input element there.
+
+**Open Question:** Should we also remove `leading` `trailing` etc props from Header? since same can be implemented inside the slot. This is what we follow for Footer.
+
+<img height="200" alt="image" src="https://github.com/razorpay/blade/assets/30949385/6867b09a-1ae2-4c4a-a977-e254b45b5282">
+
+```jsx
+<Dropdown selectionType="multiple">
+  <SelectInput label="Select Emails" />
+  <BottomSheet>
+    <BottomSheetHeader title="Do you want this report in an email?">
+      <AutoComplete label="Add receiver's email addresses" />
+    </BottomSheetHeader>
+    <BottomSheetBody>
+      <ActionList>
+        <ActionListItem title="Mumbai" value="mumbai" />
+        <ActionListItem title="Pune" value="pune" />
+        <ActionListItem title="Hyderabad" value="hyderabad" />
+      </ActionList>
+    </BottomSheetBody>
+  </BottomSheet>
+</Dropdown>
+```
+
+### With Tags Outside
+
+When possible selections are more than 11, it is advised to put tags outside of the Input.
+
+<img width="700" alt="image" src="https://github.com/razorpay/blade/assets/30949385/8906eb6c-fae0-4e99-bd3f-d7e0edc60537">
+
+We add `showTags={false}` to AutoComplete to remove tags from input. Then we can use Controlled state to show tags outside in the UI.
+
+If not tags, what to show inside AutoComplete :thinking:? Checkout this table to understand -
+
+| Code                                                            | Output                                                                                                                     |
+| --------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `<AutoComplete label="Status" showTags={false} />`              | <img width="220" alt="image" src="https://github.com/razorpay/blade/assets/30949385/c8a30845-812c-493f-a51f-97c233880960"> |
+| `<AutoComplete accessibilityLabel="Status" showTags={false} />` | <img width="200" alt="image" src="https://github.com/razorpay/blade/assets/30949385/83cf63d5-ccc4-4bb9-bf0d-ce91dc5c1b09"> |
```

Comment:
add a description that when the label is not available we show the a11y label inside input. also on parallel thought, this looks weird visually. What if we create a hidden label and assign it the a11y label here and still keep the items selected as the first option? I also understand that doesn't sets context but the tradeoff is either we show something irrelevant i.e just a11y label or we show the items selected and leave it up to consumers to use the right copywriting

Reply:
Like you said it doesn't set the context.

Imagine in this example (tbh it should just have visible label in this case but imagine such scenarios) we just show "4 items selected" but 4 items selected of what? Showing number of items selected makes sense when there is visual label to add context. 

When there is no label, there is nothing to suggest what exactly is selected. Imagine there are 3 such dropdowns side-by-side all showing "2 items selected" "4 items selected" "3 items selected" with no context.

Ideally label should always exist in such scenarios but when it isn't given, next most relevant thing to show is accessibilityLabel since that is mandatory when label is omitted.

<img width="743" alt="image" src="https://github.com/razorpay/blade/assets/30949385/b2093661-9c3d-4c88-9d0d-46bc0a78e8e4">